api-put-bucket.go

```diff
@@ -313,6 +313,92 @@ func (c Client) removeBucketLifecycle(ctx context.Context, bucketName string) er
 	return nil
 }
 
```

Comment:
Update the license header in this file and add 2020

Reply:
ok